### DISCUSSION TITLE
i send PR but no feedback

### DISCUSSION THREAD

**Discussion by vijaypotnuru** [open-sauced-craftwork/browser-extension#41] [General]
i send PR but no feedback ,do i miss any step

![FireShot Capture 004 - button add by vijaypotnuru · Pull Request #3 · open-sauced-craftwork__ - github com](https://user-images.githubusercontent.com/110402074/227452273-537e4933-787f-4818-86f8-cc1db0de6eab.png)

**Comment by mukulpadwal** [comment]
@vijaypotnuru You can tag @bdougie or @nickytonline in your pull request they will get notified.

And In the reviewers section you can select any one to review.

**Reply by bdougie** [reply]
In addition to that. You need to add a description and a screenshot to make the feedback faster.

There are over 300 submissions.

**Reply by vijaypotnuru** [reply]
i have done adding  button completely ,should add any new features to this extension?🤔

**Reply by nickytonline** [reply]
Like @bdougie said, add a description explaining things along with the screenshot you provided.

**Reply by vijaypotnuru** [reply]
Ok
